### PR TITLE
Fetch Data Online pull 2

### PR DESCRIPTION
**Pull request by vitconpc**
https://edu-redmine.framgia.vn/issues/1512
get data screen online

### CODE REVIEW

**Comment by vitconpc** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,20 +2,24 @@
...
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".screen.detail.DetailActivity"/>
     </application>
 
-</manifest>
+</manifest>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/GenresRepository.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+public class GenresRepository {
+    private static GenresRepository sInstance;
+
+    private GenresDataSource.GenresRemoteDataSource mGenresRemoteDataSource;
+
+    public GenresRepository(GenresDataSource.GenresRemoteDataSource genresRemoteDataSource) {
+        this.mGenresRemoteDataSource = genresRemoteDataSource;
+    }
+
+    public static synchronized GenresRepository getInstance(GenresDataSource.GenresRemoteDataSource genresRemoteDataSource) {
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/GenresRepository.java]: this line so long

File: app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/UpcomingRepository.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+public class UpcomingRepository {
+    private static UpcomingRepository sInstance;
+    private UpcomingDataSource.UpcomingRemoteDataSource mDataSource;
+
+    public UpcomingRepository(UpcomingDataSource.UpcomingRemoteDataSource dataSource) {
+        mDataSource = dataSource;
+    }
+
+    public static synchronized UpcomingRepository getInstance(UpcomingDataSource.UpcomingRemoteDataSource dataSource) {
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/UpcomingRepository.java]: this line so long

**Pull request merged**